primitives/core/src/registry.rs

```diff
@@ -20,11 +20,11 @@ struct Entry {
 static APP_REGISTRY: &[Entry] = &[
 	Entry {
 		app: App::ETH,
-		id: [0xdd, 0x51, 0x4b, 0xaa, 0x31, 0x7b, 0xf0, 0x95, 0xdd, 0xba, 0x2c, 0x0a, 0x84, 0x77, 0x65, 0xfe, 0xb3, 0x89, 0xc6, 0xa0],
+		id: [252, 151, 166, 25, 125, 201, 11, 239, 107, 190, 253, 103, 39, 66, 237, 117, 233, 118, 133, 83],
```

Comment:
Why app addresses are changed?

Reply:
These are testnet addresses. All addresses are testnet addresses.

Reply:
I suppose you mean the local testnet. But in this case, the node will not be working with Kovan.
Are these values parameterized using environment?

Reply:
Yes, I mean the local testnet. Yes, the node will not work with Kovan ootb. You will need to set variables as per your deployment.
No. Not yet.

Reply:
Ok please revert the value to match public testnet's Kovan address

Reply:
What about the MNG contract address? That will require a deployment of the ERC20 MNG token on Kovan. Should I leave that as it is (setup for the local testnet)?